org.signal.Signal-Beta.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id": "org.signal.Signal-Beta",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "signal",
+    "separate-locales": false,
+    "rename-desktop-file": "signal-desktop-beta.desktop",
+    "rename-icon": "signal-desktop-beta",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=org.kde.StatusNotifierItem-2-1"
+    ],
+    "modules": [
+        {
+            "name": "signal-desktop-beta",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar ox signal-desktop-beta_amd64.deb",
+                "tar xf data.tar.xz",
+                "mv 'opt/Signal Beta' opt/Signal-Beta",
+                "cp --recursive --preserve usr/* opt/* /app",
+                "rm -rf /app/share/icons/hicolor/1024x1024/ /app/Signal-Beta/chrome-sandbox",
```

Comment:
Why do you remove `chrome-sandbox` only to create it next? Please drop all chrome-sandbox` related commands.

Reply:
I guess if you just copy-pasted signal manifest it would save you much of the troubles.

Reply:
Did you forget your comment? https://github.com/refi64/zypak#common-problems

>  I guess if you just copy-pasted signal manifest it would save you much of the troubles.

Could you please read the description of this PR?

Reply:
When I wrote my first comment I didn't know you broke `zypak` by yourself by removing `chrome-sandbox`. Obviously stopping doing that is better solution than re-creating it. The description says that difference is `--no-sandbox` which was wrong solution for problem that you created due to not simply copying signal manifest which was known to work. In general I think this should go to `beta` branch of signal rather than separate thing that reinvents the wheel.

Reply:
Yeah, I don't get why you remove it. There's plenty of Electron apps that do just fine with regular chrome-sandbox.